static/bidder-info/onetag.yaml

```diff
@@ -19,4 +19,7 @@ userSync:
   iframe:
     url: "https://onetag-sys.com/usync/?redir={{.RedirectURL}}&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}"
     userMacro: "${USER_TOKEN}"
-endpointCompression: "GZIP"
\ No newline at end of file
+  redirect:
+    url: "https://onetag-sys.com/usync/?tag=img&redir={{.RedirectURL}}&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}"
+    userMacro: "${USER_TOKEN}"
```

Comment:
redirect url is working. user id was appended at the end of url

`https://onetag-sys.com/usync/?tag=img&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Donetag%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%22%24%7BUSER_TOKEN%7D%22&gdpr=&gdpr_consent=&us_privacy=`

<img width="907" alt="image" src="https://github.com/prebid/prebid-server/assets/24757781/bd484ee5-c153-4215-8fbe-b51dd9cd070e">